pkg/volume/local/local.go

```diff
@@ -204,8 +221,25 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
-		return nil
+	refs, err := mount.GetMountRefsByDev(m.mounter, m.globalPath)
+	if fsGroup != nil {
```

Comment:
the reason "refs, err := mount.GetMountRefsByDev(m.mounter, m.globalPath)" is done before "if"  is that I did not want it variable refs to be in if scope, so it would be visible at the end of this function for checking whether this volume has already been mounted or not.